packages/efate/src/fixture.ts

```diff
@@ -1,24 +1,17 @@
 import debugGenerator from 'debug';
 import merge from 'lodash.merge';
-import type { PartialDeep } from 'type-fest';
 
-import { BuilderGeneratorFunction } from './types';
+import { FixtureFieldBuilder, FixtureOverrider, FixturesArrayBuilder } from './types';
 import { isFunction } from './utils';
 
 const debug = debugGenerator('efate:fixture-factory');
 
-type OverridesFn<T> = (fixture: T) => void;
-type Overrides<T> = PartialDeep<T> | OverridesFn<T>;
```

Comment:
PartialDeep is necessary so you can define partial nested types.  See longer description with PR comment